boards/weact_f401ccu6/layout.ld

```diff
@@ -1,13 +1,13 @@
 /* Memory layout for the STM32F401CCU6
- * rom = 256KB (LENGTH = 0x00040000)
- * kernel = 128KB
- * user = 128KB
- * ram = 64KB */
```

Comment:
The debug version of the kernel doesn't fit in the allocated space. Should the kernel space be increased to allow debug version flashing?